packages/@lwc/module-resolver/README.md

```diff
@@ -21,10 +21,156 @@ const result = resolveModule('x/foo', './index.js');
 console.log(result);
 ```
 
+If in the middle of the module resolution the resolver processes an invalid configuration the resolver throws an error with the `LWC_CONFIG_ERROR` error code. If a module can't be resolved from the given importer the invocation throws an error with the `NO_LWC_MODULE_FOUND` error code.
+
+```js
+import { resolveModule } from '@lwc/module-resolver';
+
+let result;
+try {
+    result = resolveModule('x/foo', './index.js');
+} catch (err) {
+    if (err.code === 'LWC_CONFIG_ERROR') {
+        console.error(`The request module can't be resolved due to an invalid configuration`, err);
+    } else if (err.code === 'NO_LWC_MODULE_FOUND') {
+        console.error(`The requested module doesn't exists`, error);
+    } else {
+        throw err;
+    }
+}
+
+console.log(result);
+```
+
 **Parameters:**
 
 -   `specifier` (string, required): The module specifier that should be resolved.
 -   `dirname` (string, required): The directory from where the specifier should be resolved.
```

Comment:
```suggestion
-   `dirname` (string, required): The module specifier is resolved from this directory.
```

Reply:
I don't think we should use the active voice to describe this argument.